modules/commandModule/commandFns.py

```diff
@@ -31,4 +32,30 @@ def write_pigo(newPigo):
 	return
 #if nothing has changed, return the dictionary that was passed in
     else:
-        return
\ No newline at end of file
+        return
+
+def read_pogi() -> dict:
```

Comment:
It would be really useful if we could have a way to tell if the pogi has been updated, since we don't want to shove redundant data into our pipelines. Here's my idea for implementing this:

1) Once you've run everything in lines 53-60, save that pogi as a json called last_pogi
2) Everytime this function is run, first have it check if a last_pogi.json exists. If it does, load it, and using a similar algorithm to what is used in write_pogi to check if anything has been changed. 
3) If nothing has been changed, return `False, pogiData`, if something has been changed return `True, pogiData`.